vitality/workout.py

```diff
@@ -0,0 +1,16 @@
+class Workout:
+    def __init__(self, w_name, difficulty, w_id, about, exp_rewards):
+        self.w_name = w_name
+        self.difficulty = difficulty
+        self.w_id = w_id
+        self.about = about
+        self.exp_rewards = exp_rewards
```

Comment:
`w_id` would best be `id`. Rename.  

In the parameter list: 
`about` should equal None by default. 
`exp_rewards` should equal None by default. 

Once these and the below lines are fixed I'll approve.

Reply:
Also `w_name` should just be `name`